### ISSUE TITLE
vars not being loaded in rails console

### ISSUE THREAD

**Issue by aristizabal95** [rbenv/rbenv-vars#35]
Hello, I know there are a lot of threads and issues about rbenv vars not being loaded on database.yml and everything. Sadly, I haven't found any solution for my specific case.

I've updated rbenv
I've installed the plugin as explained on the README.
I've created the `.rbenv-vars` file on my app's root directory.
I've tested the installation by running `rbenv vars` and received the expected output.

Still, when I do `rails c` and run `ENV['SECRET_KEY_BASE']` or any other key set inside the `rbenv-vars` file, I receive `nil`. I can still access any environment variable set by bash, like `ENV['OLDPWD']`

Basically, I can see rbenv-vars is working correctly, but my app cannot access any of the variables inside the `.rbenv-vars` file.

Not sure if this adds any info, but running `printenv` doesn't show any of the variables set through rbenv vars.

**Comment by juansecaro** [comment]
Same problem

**Comment by mislav** [comment]
Hi, I'm sorry for the late reply. From what you've shared I can't tell why the variables aren't loaded, but I just tried to reproduce and I couldn't. Please make sure that rbenv is set up properly and that you can verify a variable being loaded in an isolated Ruby process:

```sh
ruby -e 'p ENV["MY_VAR"]'
```

It could be that your `rails` process was started without involvement of rbenv, which would lead to rbenv-vars being skipped.
